python/lsst/ts/MTDome/mock_llc/amcs.py

```diff
@@ -203,11 +206,25 @@ async def park(self, start_tai):
             the real dome, this should be the current time. However, for unit
             tests it can be convenient to use other values.
         """
-        self.status = MotionState.PARKING
+        self.status = LlcMotionState.PARKING
         self.position_commanded = PARK_POSITION
         self.duration = self.azimuth_motion.park(start_tai)
         return self.duration
 
+    async def go_stationary(self, start_tai):
+        """Set the dome to stationary state by stopping the motion.
```

Comment:
Brakes? Locking pins? The doc strings in the mock controller are very clear; please consider copying them to the mock-subsystem methods.

Reply:
Yes, I was being sloppy again. Done.